File: src/main/java/com/github/nsc/de/shake/parser/node/logical/LogicalFalseNode.java
package com.github.nsc.de.shake.parser.node.logical;

import com.github.nsc.de.shake.util.characterinput.position.PositionMap;

public class LogicalFalseNode extends LogicalNode {

    public LogicalFalseNode(PositionMap map) {
        super(map);
    }

    @Override
    public String getOperator() {
        return null;
    }

    @Override
    public String toString() {
        return "false";
    }
}

File: parser/src/main/kotlin/com/github/nsc/de/shake/parser/node/logical/LogicalFalseNode.kt
package com.github.nsc.de.shake.parser.node.logical

import com.github.nsc.de.shake.util.characterinput.position.PositionMap

class LogicalFalseNode(map: PositionMap) : LogicalNode(map) {
    override val operator: String?
        get() = null

    override fun toString(): String {
        return "false"
    }
}